.gitmodules

```diff
@@ -14,3 +14,6 @@
 	path = kubernetes
 	url = https://github.com/openshift/kubernetes
 	branch = release-4.6
+[submodule "promu"]
```

Comment:
Please add the commands you executed to add the submodule

Reply:
Where should this be added? To the commit message?

Reply:
yes to the commit message please